sys/bindgen.rs

```diff
@@ -34,8 +34,12 @@ pub fn generate_bindings(api_dir: &Path) {
         .clang_arg(format!("-I{}/include", api_dir.display()))
         .trust_clang_mangling(false)
         .default_non_copy_union_style(bindgen::NonCopyUnionStyle::ManuallyDrop)
-        .blocklist_type("widechar")
-        .new_type_alias("CommandList")
```

Comment:
- `widechar` was not used anywhere;
- newtyping `CommandList` is useless when all other types are plain typedefs. This needs a more generalized approach - later.